pyerrors/input/openQCD.py

```diff
@@ -343,3 +345,209 @@ def _read_array_openQCD2(fp):
 
     arr = _parse_array_openQCD2(d, n, size, tmp, quadrupel=True)
     return {'d': d, 'n': n, 'size': size, 'arr': arr}
+
+
+def read_qtop(path, prefix,c, dtr_cnfg = 1,**kwargs):
+    """Read qtop format from given folder structure.
+
+    Parameters
+    ----------
+    path:
+        path of the measurement files
+    prefix:
+        prefix of the measurement files, e.g. <prefix>_id0_r0.ms.dat
+    c:
+        ???
+    dtr_cnfg:
+        ???
+    target: int
+        specifies the topological sector to be reweighted to (default 0)
+    full: bool
+        if true read the charge instead of the reweighting factor.
+    version: str
+        version string of the openQCD (sfqcd) version used to create the ensemble
+    steps: int
+        step size of measurements
+    L: int
+        spatial length of the lattice in L/a. HAS to be set if version != sfqcd, since openQCD does not provide this in the header
+    r_start: list
+        offset of the first ensemble, making it easier to match later on with other Obs
+    r_stop: list
+        last ensemble that needs to be read
+    r_meas_start: list
+        offset of the first measured ensemble, if there is any
+    files: list
+        specify the exact files that need to be read from path, pratical if e.g. only one replicum is needed
+    names: list
+        Alternative labeling for replicas/ensembles. Has to have the appropriate length
+    """
+    #dtr_cnfg = 4# was ist das denn hier?
+    #one could read L from the header in case of sfQCD
+    #c = 0.35
+    known_versions = ["1.0","1.2","1.4","1.6","2.0", "sfqcd"]
+    version = "1.2"
+    if "version" in kwargs:
```

Comment:
I would propose to make `version` a parameter in line 350. This would make lines 388-390 redundant.

Reply:
Okay, version is now a parameter with default value `"1.2"`